src/drivers/drv_pwm_output.h

```diff
@@ -424,20 +416,20 @@ __EXPORT extern servo_position_t up_pwm_servo_get(unsigned channel);
 /**
  * Intialise the Dshot outputs using the specified configuration.
  *
- * @param	channel_mask	Bitmask of channels (LSB = channel 0) to enable.
- *			This allows some of the channels to remain configured
- *			as GPIOs or as another function.
- * @param	dshot_pwm_freq is frequency of DSHOT signal. Usually DSHOT1200, DSHOT600, DSHOT300 or DSHOT150
- * @return	OK on success.
+ * @param channel_mask		Bitmask of channels (LSB = channel 0) to enable.
+ *				This allows some of the channels to remain configured
+ *				as GPIOs or as another function.
+ * @param dshot_pwm_freq	Frequency of DSHOT signal. Usually DSHOT150, DSHOT300, DSHOT600 or DSHOT1200
+ * @return OK on success.
  */
 __EXPORT extern int up_dshot_init(uint32_t channel_mask, unsigned dshot_pwm_freq);
 
 /**
  * Set the current dshot throttle value for a channel (motor).
  *
  * @param channel	The channel to set.
- * @param throttle	The output dshot throttle value in [0, 1999 = DSHOT_MAX_THROTTLE].
- * @param telemetry If true, request telemetry from that motor
+ * @param throttle	The output dshot throttle value in [0 = DSHOT_DISARM_CMD, 1999 = DSHOT_MAX_THROTTLE].
```

Comment:
One last thing: you forgot to update this one. However I'd rather not mention it, as 0 isn't actually set as throttle (`up_dshot_motor_command` is used instead for disarming).
```suggestion
 * @param throttle	The output dshot throttle value in [0, 1999 = DSHOT_MAX_THROTTLE].
```

Reply:
Thanks! Nice catch!  I added `1 = DSHOT_MIN_THROTTLE` as well.